source/server/admin/stats_handler.cc

```diff
@@ -96,20 +96,21 @@ Admin::RequestPtr StatsHandler::makeRequest(AdminStream& admin_stream) {
     server_.flushStats();
   }
 
+  bool active_mode;
```

Comment:
move declaration into both branches and make it const?

Reply:
I had it as `constexpr` and it broke the build in some variation/compiler, so I made it this. I actually like this one better because it's clear that the value is used in both branches, so it must exist unconditionally. It's possible the compiler will still be grumpy about `const` (not `constexpr`), but not sure. I can try that if you want.

Reply:
What you have is fine.